CDCgov/cdc-open-viz
# Feature/dev 8607

## Testing Steps
Open Dashboard component.Click Tab "Dashboard Description" 
On new Window we will see an icon GEAR on the corner as shown on the image.


<img width="422" alt="Screenshot 2024-08-02 at 16 34 33" src="https://github.com/user-attachments/assets/b42a3290-eea8-4d83-a3ec-9b2d5b752595">

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
